src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -83,7 +83,7 @@ const OfflineKeyShareGeneration = () => {
   };
 
   const goToChangeOperators = () => {
-    navigate(-4);
+    navigate(-2);
```

Comment:
What is the meaning of -2 when navigating?

Reply:
routing -2 steps return us to choose operators